src/client/src/orangehrmCorporateDirectoryPlugin/components/SummaryCardDetails.vue

```diff
@@ -0,0 +1,263 @@
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <oxd-sheet :gutters="false" class="orangehrm-directory-card" type="white">
+    <div
+      class="orangehrm-directory-card-top"
+      @click="$emit('hide-details', false)"
+    >
+      <oxd-icon name="arrow-right"></oxd-icon>
+    </div>
+    <div class="orangehrm-directory-card-header">
+      <oxd-text type="card-title">
+        {{ employeeName }}
+      </oxd-text>
+    </div>
+    <profile-picture :id="id"></profile-picture>
+    <div class="orangehrm-directory-card-header">
+      <oxd-text type="toast-title">
+        {{ employeeDesignation }}
+      </oxd-text>
+    </div>
+    <div class="orangehrm-directory-card-body">
+      <span class="orangehrm-directory-card-icon">
+        <oxd-icon name="geo-alt-fill"></oxd-icon>
+      </span>
+      <span>
+        <div class="orangehrm-directory-card-subunit">
+          <oxd-text type="toast-message">
+            {{ employeeSubUnit }}
+          </oxd-text>
+        </div>
+        <div class="orangehrm-directory-card-location">
+          <oxd-text type="toast-message">
+            {{ employeeLocation }}
+          </oxd-text>
+        </div>
+      </span>
+    </div>
+    <div class="orangehrm-directory-card-rounded-body">
+      <div class="orangehrm-directory-card-icon">
+        <oxd-icon-button display-type="success" name="telephone-fill">
+        </oxd-icon-button>
+      </div>
+      <div class="orangehrm-directory-card-icon">
+        <oxd-icon-button display-type="danger" name="mailbox">
+        </oxd-icon-button>
+      </div>
+    </div>
+    <div
+      class="orangehrm-directory-card-hover"
+      @mouseleave="showTelephoneClip = false"
+      @mouseover="showTelephoneClip = true"
+    >
+      <div class="orangehrm-directory-card-hover-body">
+        <oxd-text type="toast-message">{{ $t('Work Telephone') }}</oxd-text>
+        <oxd-text type="toast-title"> {{ employeeWorkTelephone }}</oxd-text>
+      </div>
+      <div
+        class="orangehrm-directory-card-hover-body orangehrm-directory-card-icon"
+      >
+        <oxd-icon v-show="showTelephoneClip" name="clipboard-check"></oxd-icon>
+      </div>
+    </div>
+    <oxd-divider></oxd-divider>
+    <div
+      class="orangehrm-directory-card-hover"
+      @mouseleave="showEmailClip = false"
+      @mouseover="showEmailClip = true"
+    >
+      <div class="orangehrm-directory-card-hover-body">
+        <oxd-text type="toast-message">{{ $t('Work Email') }}</oxd-text>
+        <oxd-text type="toast-title"> {{ employeeWorkEmail }}</oxd-text>
+      </div>
+      <div
+        class="orangehrm-directory-card-hover-body orangehrm-directory-card-icon"
+      >
+        <oxd-icon v-show="showEmailClip" name="clipboard-check"></oxd-icon>
+      </div>
+    </div>
+    <oxd-divider></oxd-divider>
+    <div class="orangehrm-directory-card-qrcode">
```

Comment:
Can we remove this for now. QR code is not of the scope and it should be generated not be static

Reply:
Removed